### PR TITLE
fix for #1545

### LINKED ISSUE #1545: remote code execution
NCSA security has discovered a remote code execution that will need to be patched. For more details contact me.

### PR DESCRIPTION
**Pull request by robkooper**
Fix remote code execution in dataset.php, this is only executable by providing a malicious url, and not from the web interface.

## Types of changes

- [x] Bug fix (non-breaking change which fixes #1545) 
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] My change requires a change to the documentation.
- [x] I have updated the CHANGELOG.md.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Comment by robkooper** [comment]
This is deployed at http://pecan.ncsa.illinois.edu, and can be tested at http://pecan.ncsa.illinois.edu/pecan/08-finished.php?workflowid=99000000002

**Comment by tonygardella** [comment]
Assigned myself because I'm matched with Rob this week for code review.

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,6 +7,9 @@ For more information about this file see also [Keep a Changelog](http://keepacha
 
 ## [Unreleased]
 
+### Fixes
+- Fixed remote code execution #1545
+
 ## [1.5.10] - Prerelease
```
Comment (tonygardella) [inline CHANGELOG.md]: Could you change this to: `## [1.5.0]- 2017-07-13`

**Comment by tonygardella** [comment]
If you don't have time to make that edit to the change log I can do that later. I'd rather this get merged asap.

**Pull request merged**